### PR TITLE
Support code_version from Rollbar

### PR DESCRIPTION
**Pull request by erichard**

### CODE REVIEW

**Comment by megazoll** [comment]
Please fix the tests.

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -22,6 +22,8 @@ public function getConfigTreeBuilder()
                 ->scalarNode('environment')
                     ->defaultValue('%kernel.environment%')
                 ->end()
+                ->scalarNode('code_version')
```
Comment (megazoll) [inline DependencyInjection/Configuration.php]: This option should have default value if we want keep bc.

**Comment by megazoll** [comment]
Thanks!

**Comment by erichard** [comment]
@megazoll Could you tag a version ?

**Comment by megazoll** [comment]
Tagged as v1.3.1

**Pull request merged**